cmd/osm-controller/validate.go

```diff
@@ -24,10 +24,6 @@ func validateCLIParams() error {
 		return errors.Errorf("Please specify the webhook configuration name using --webhook-config-name")
 	}
 
-	if caBundleSecretName == "" {
-		return errors.Errorf("Please specify the CA bundle secret name using --ca-bundle-secret-name")
-	}
-
 	if caBundleSecretName == "" {
 		return errors.Errorf("Please specify the CA bundle secret name using --ca-bundle-secret-name containing the cert-manager CA at 'ca.crt'")
```

Comment:
@SanyaKochhar the log message you removed is accurate while the one left here is referring to cert-manager which is not correct. Would you mind swapping those?